### PR TITLE
Changes GalleryActivity to belong_to and set_app to use user_level.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: dashboard/app/models/gallery_activity.rb [hunk 1]
```diff
@@ -26,12 +26,18 @@ class GalleryActivity < ActiveRecord::Base
...
 
   before_save :set_app
 
   def set_app
-    return unless activity
-    self.app = activity.try(:level).try(:game).try(:app)
+    if user_level
+      self.app = user_level.level.try(:game).try(:app)
+      return
+    end
```
Comment (mehalshah) [inline dashboard/app/models/gallery_activity.rb]: how about 
if user_level
self.app = ...
else if level_source
self.app = ...
end

Then you can skip the return on 36

Comment (ashercodeorg) [reply dashboard/app/models/gallery_activity.rb]: Good call. Done.

**Comment by ashercodeorg** [comment]
Nope, I don't understand the CircleCI failures. But I'm seeing similar on all my other branches, so I don't think we should gate on that.

**Comment by mehalshah** [comment]
Ship it

**Pull request merged**